cards/multilang-product-prominentvideo/component.js

```diff
@@ -0,0 +1,77 @@
+{{> cards/card_component componentName='multilang-product-prominentvideo' }}
+
+class multilang_product_prominentvideoCardComponent extends BaseCard['multilang-product-prominentvideo'] {
+  constructor(config = {}, systemConfig = {}) {
+    super(config, systemConfig);
+  }
+
+  /**
+   * This returns an object that will be called `card`
+   * in the template. Put all mapping logic here.
+   *
+   * @param profile profile of the entity in the card
+   */
+  dataForRender(profile) {    
+    profile.featuredMessage = {
+      description: 'subtitle'
+    }
+    
+    const videoUrl = profile.videos && profile.videos[0]?.video?.url;
+    const youtubeVideoId = videoUrl?.split('watch?v=')[1];
+    const youtubeUrl = youtubeVideoId ? 'https://www.youtube.com/embed/' + youtubeVideoId : null;
+    const vimeoUrl = profile.c_vimeo;
+
+    return {
+      title: profile.name, // The header text of the card
+      url: profile.landingPageUrl, // If the card title is a clickable link, set URL here
+      target: '_top', // If the title's URL should open in a new tab, etc.
+      titleEventOptions: this.addDefaultEventOptions(),
+      subtitle: profile.featuredMessage.description, // The sub-header text of the card
```

Comment:
Will this always just be hard-coded to `'subtitle'`?

Reply:
Oops, that was for testing and I forgot to remove it